v22.2/cdc-transformations.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Change Data Capture Transformations
+summary: Filter the changefeed data that emits to the changefeed sink.
+toc: true
+docs_area: stream
+---
+
+{% include feature-phases/preview.md %}
+
+{% include_cached new-in.html version="v22.2" %} Change data capture transformations allow you to define the change data emitted to your sink when you create a changefeed. The expression [syntax](#syntax) provides a way to select columns and apply filters to further restrict or transform the data in your [changefeed messages](changefeed-messages.html).  
+
+You can use CDC transformations to do the following:
+
+- Filter out specific rows and columns from changefeed messages to decrease the load on your downstream sink and support outbox workloads.
+- Modify data before it emits to reduce the time and operational burden of filtering or transforming data downstream.
+- Stabilize or customize the schema of your changefeed messages for increased compatibility with external systems.
+
+You can use any CockroachDB-supported SQL expression syntax that is not listed in [limitations](#limitations) to build a changefeed query.
+
+See the [Examples](#examples) section for further use cases.
+
+## Syntax
+
+There are two possible components to CDC transformations:
+
+- _Projections_ will select the columns that you want to emit data from. 
+- _Predicates_ will restrict the selected column change data based on the filters you apply.
+
+~~~ sql
+CREATE CHANGEFEED [INTO sink] [WITH options] AS SELECT projection FROM table [WHERE predicate];
+~~~
+
+Parameter        | Description
+-----------------+---------------
+`sink`             | Specify the sink URL to emit change data to. See [Changefeed Sinks](changefeed-sinks.html) for a list of supported sinks. It is also possible to run a changefeed without a sink `CREATE CHANGEFEED WITH...`, which will send changes to the active SQL session.
+`options`          | Set options on the changefeed. See the [Options](create-changefeed.html#options) table for a full list.
+`projection`       | Select the columns from which to emit data.
+`table`            | Define the table to which the columns belong.
+`predicate`        | Apply optional filters with a `WHERE` clause.
+
+For a SQL diagram of the CDC transformation syntax, see the [`CREATE CHANGEFEED`](create-changefeed.html#synopsis) page.
+
+## Limitations
+
+{% include {{ page.version.version }}/known-limitations/cdc-transformations.md %}
+
+## CDC transformation function support
+
+You can use the following functions in CDC transformation queries:
+
+- "Immutable" as marked on the [Functions and Operators page](functions-and-operators.html).
```

Comment:
nit: recommend starting with the noun in focus (this also matches phrasing under `### Unsupported functions`): 

> Functions marked as "Immutable" on the [Functions and Operators page](functions-and-operators.html)